.github/workflows/unit_race.yml

```diff
@@ -130,6 +130,7 @@ jobs:
         # which musn't be more than 107 characters long.
         export VTDATAROOT="/tmp/"
         export NOVTADMINBUILD=1
+        export VTEVALENGINETEST="-1"
```

Comment:
One last nit: -1 is a weird value for this env variable. Should have been 0 for false, and 1 for true.

Reply:
Okay, I'll change it 👍